script/input.js

```diff
@@ -0,0 +1,92 @@
+const instructors = [
+  {
+    image: '../images/milla.jpg',
+    name: 'Million Ayalew',
+    job: 'Software Engineering graduate at ',
+    profile:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Et veritatis harum nesciunt nihil quae sit aliquam ab doloribus? Eum, ipsam!',
+  },
+  {
+    image: '../images/Ag.jpg',
+    name: 'Agmas Getnet',
+    job: 'Software Engineer at Ministry of Innovation and TEchnology ',
+    profile:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Et veritatis harum nesciunt nihil quae sit aliquam ab doloribus? Eum, ipsam!',
+  },
+  {
+    image: '../images/Ad.jpg',
+    name: 'Addis Tsega',
+    job: 'Graduate of Software Engineering @ AASTU',
+    profile:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Et veritatis harum nesciunt nihil quae sit aliquam ab doloribus? Eum, ipsam!',
```

Comment:
- [OPTIONA] I recommend fixing the image issue, so that in can appear in you instructor section as well.  

![image](https://user-images.githubusercontent.com/43877689/181806363-27410e37-68b4-4294-ac5b-e6a2f79e9714.png)